src/debugger/appWorker.ts

```diff
@@ -92,8 +95,12 @@ export class SandboxedAppWorker {
 
     private initializeSandboxAndContext(scriptToRunPath: string): void {
         let scriptToRunModule = new Module(scriptToRunPath);
+        scriptToRunModule.paths = Module._nodeModulePaths(path.dirname(scriptToRunPath));
 
         this.sandbox = {
+            __debug__: {
+                require: (filePath: string) => scriptToRunModule.require(filePath),
```

Comment:
Please add some context as comments explaining that we need this for realm, links to the issue, etc...